.github/workflows/test-openmcyclus.yml

```diff
@@ -18,35 +16,25 @@ jobs:
           miniforge-variant: Mambaforge
           miniforge-version: latest
           activate-environment: openmcyclus-env
-          environment-file: environment.yml
           use-mamba: true
 
       - name: Conda config
         run: |
           conda config --env --set pip_interop_enabled True 
       
-      - name: Update Environment
+      - name: Install dependencies
         run: |
-          pip install pytest
-          mamba env update -n openmcyclus-env -f environment.yml
+          mamba install -y cycamore openmc scipy=1.11 pytest
```

Comment:
Beautiful!

Reply:
When I run the CI with the mamba build (see run 129)[https://github.com/arfc/openmcyclus/actions/runs/9715733837/job/26817800858], I think it finds cyclus. So this build configuration will stay.